src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -124,14 +129,36 @@ async function getLandoApplication( lando: Lando, instancePath: string ): Promis
 		throw new Error( DEV_ENVIRONMENT_NOT_FOUND );
 	}
 
-	const app = lando.getApp( instancePath );
-	addHooks( app, lando );
-	appMap.set( instancePath, app );
+	let app;
 
-	if ( ! app.initialized ) {
+	try {
+		app = lando.getApp( instancePath );
+		addHooks( app, lando );
 		await app.init();
+	} catch ( error ) {
+		debug( 'Error initializing Lando app', error );
+		console.warn( chalk.yellow( 'There was an error initializing Lando, trying to recover...' ) );
```

Comment:
Not sure if we would ever need it, but should we take a back-up if the failed yml file?

Reply:
I kinda like the idea, better for debugging too.